src/Forms/GoToLine.cs

```diff
@@ -85,10 +85,7 @@ private void TextBox1KeyDown(object sender, KeyEventArgs e)
                     e.KeyCode == Keys.End ||
                     (e.KeyValue >= 96 && e.KeyValue <= 105))
                 {
-                }
-                else if (e.KeyData == (Keys.Shift | Keys.Home) || e.KeyData == (Keys.Shift | Keys.End) ||

```

Comment:
This code isn't needed 'cause it would never run... the first condition will cover it